### DISCUSSION TITLE
Imported variables in style files failing

### DISCUSSION THREAD

**Discussion by Draugsvoll** [vitest-dev/vitest#3000] [Q&A]
Im working in a react+typescript project, and when running tests its not able to resolve imported variables in my Style files. For example im importing a color array from a .ts file and it ends up undefined. How can i resolve this?

**Comment by sheremet-va** [comment]
CSS is not processed by default in Vitest. Enabled `test.css` in your config file.

**Comment by Draugsvoll** [comment]
Can you be more specific? I tried adding 'css: true' and include my files containing the color-array etc in the 'setupFiles' field, still same result

**Reply by sheremet-va** [reply]
https://vitest.dev/config/#css
